src/portable/renesas/usba/dcd_usba.c

```diff
@@ -547,7 +552,8 @@ void dcd_set_address(uint8_t rhport, uint8_t dev_addr)
 void dcd_remote_wakeup(uint8_t rhport)
 {
   (void)rhport;
-  /* TODO */
+  USB0.DVSTCTR0.BIT.WKUP = 1;
+  while (USB0.DVSTCTR0.BIT.WKUP) ;
```

Comment:
I think we don't really need to blocking wait here,which could take a few ms, just return. We will send resume event later on with SOF walkaround. The usbd has its own suspended status bit, and shouldn't do any transfer until RESUME event is reported by dcd.

Reply:
OK, I will remove this statement.

Reply:
The statement is removed at 1ff3b76.